sig-advocacy-and-outreach/README.md

```diff
@@ -0,0 +1,38 @@
+# Advocacy and Outreach Special Interest Group
+
+The Advocacy and Outreach SIG aims to grow the community in an innovative way. 
+We discuss advocacy, outreach, events, social media promotion, 
+and attract new people to join KubeSphere community.
+
+## Members
+
+- Pengfei Zhou ([@FeynmanZhou](https://github.com/FeynmanZhou)), Lead
+- Shaowen Chen ([@shaowenchen](https://github.com/shaowenchen)), Member
+- Honglei ([@shenhonglei](https://github.com/shenhonglei/)), Member
+- webup ([@webup](https://github.com/webup/)), Member
+- faweizhao26 ([@faweizhao26](https://github.com/faweizhao26/)), Member
+- Rick ([@linuxsuren](https://github.com/linuxSuRen/)), Member
```

Comment:
I would recommend Rick to be a lead in this SIG, SIG-Advocacy-and-Outreach was proposed by Rick : )

Reply:
Pengfei has done a lot of community management work before. Of course, I'd be happy to be the co-leader of this SIG.

Reply:
Please also add yourself, thanks.

Reply:
Done